boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -127,11 +127,6 @@ ifeq ($(origin TESTTARGETS), undefined)
 TESTTARGETS := $(shell ${GOENV} go list -e ./... | grep -E -v "/(vendor)/" | grep -E -v "/(test/e2e)/")
 endif
 
-# If for any reason we've made it this far and TESTTARGETS is still empty, fail early.
-ifeq ($(TESTTARGETS),)
-$(error TESTTARGETS is empty)
-endif
```

Comment:
This would cause any failure of `go list` to cause any target to fail, even if it didn't rely on `go list` working. This can be seen in https://ci.int.devshift.net/job/openshift-aws-account-operator-gh-build-master/524/console